### PR TITLE
Added in scenarios and testing

### PR DESCRIPTION
**Pull request by rloomba**

### CODE REVIEW

File: scenarios/credit_failed_state/request.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%= boiler_plate %>
+marketplace = Balanced::Marketplace.my_marketplace
+
+bank_account_info = {
+<%="".tap { |s| request['bank_account'].each {|k, v| s << ":#{k} => '#{v}',"} }%>
```
Comment (mjallday) [inline scenarios/credit_failed_state/request.rb]: should this be a function? it looks like you're doing the same string formatting on every scenario with params

Comment (rloomba) [inline scenarios/credit_failed_state/request.rb]: Yeah, it probably should be. I'll abstract this out into the calling script that renders this template. It should then be something similar to `bank_account_info = { params_to_hash(request['bank_account']} }`

**Comment by mjallday** [comment]
Does this fix #80? What needs to happent merge this in?

**Comment by rloomba** [comment]
@mjallday Yeah this fixes #80. After this is merged in, we need to close this pull request: https://github.com/balanced/balanced-docs/pull/88

I'm currently working on changing the docs over to use the customer resource. I might have more scenarios to add to the Ruby client. Once I'm done, I'll update this and the node client and merge.

**Pull request merged**